flathub.json

```diff
@@ -1,3 +0,0 @@
-{
-    "skip-appstream-check": true
```

Comment:
You should instead put `"skip-icons-check": true`

Reply:
Ah, nice! Thank you for the hint!